README.md

```diff
@@ -1,17 +1,25 @@
 # TrustlySDK
 
-[![CI Status](https://img.shields.io/travis/Marcos Rivereto/TrustlySDK.svg?style=flat)](https://travis-ci.org/Marcos Rivereto/TrustlySDK)
-[![Version](https://img.shields.io/cocoapods/v/TrustlySDK.svg?style=flat)](https://cocoapods.org/pods/TrustlySDK)
-[![License](https://img.shields.io/cocoapods/l/TrustlySDK.svg?style=flat)](https://cocoapods.org/pods/TrustlySDK)
-[![Platform](https://img.shields.io/cocoapods/p/TrustlySDK.svg?style=flat)](https://cocoapods.org/pods/TrustlySDK)
-
-## Example
+[![License](https://badgen.net/badge/license/MIT/blue?icon=swift)](https://cocoapods.org/pods/TrustlySDK)
+[![Swift Version](https://badgen.net/badge/swift/5/orange?icon=swift)](https://cocoapods.org/pods/TrustlySDK)
+[![Platform](https://badgen.net/badge/iOS/v12/green?icon=swift)](https://cocoapods.org/pods/TrustlySDK)
+[![Cocoapods Version](https://badgen.net/badge/cocoapods/v1.12.0/blue?icon=swift)](https://cocoapods.org/pods/TrustlySDK)
+[![Pod Version](https://badgen.net/badge/pod/v3.0.0/yellow?icon=swift)](https://cocoapods.org/pods/TrustlySDK)
+[![Swift Package Maager](https://badgen.net/badge/spm/available/green?icon=swift)](https://cocoapods.org/pods/TrustlySDK)
 
+## Example App
```

Comment:
Should we simply remove example apps from this public version? Since we also have an official example I think this section should just direct users to that github.com/TrustlyInc/trustly-ios-example.

Reply:
Ok, fair enough. In that case let's keep it and ship it! LGTM!